src/utils/oktaConfig.js

```diff
@@ -1,5 +1,5 @@
 const config = {
-  issuer: process.env.REACT_APP_OKTA_ISSUER_URI,
+  issuer: process.env /**TO-DO: Implement Auth0 */,
```

Comment:
Seeing as how okta will no longer be needed, if all references to okta have otherwise been removed, this file can safely be removed as well. From what I remember, there's no additional config file needed for Auth0 - it's all configured in the provider (wrapping the app) and on the Auth0 dashboard.

Reply:
Oh okay, cool